core/app.go

```diff
@@ -150,8 +150,10 @@ func (a *App) Run() {
 	if 1 == os.Getpid() {
 		reapChildren()
 	}
+
+	a.ControlServer.Start(a)
```

Comment:
If we move this outside the loop then don't we fail to restart the control server on a reload?